### PR TITLE
Add a preset for Cake.Recipe based builds

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

File: cake-recipe.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "description": "Use Renovate configuration for Cake.Recipe based builds",
+    "extends": [
+        "github>cake-contrib/renovate-presets"
+    ],
+    "packageRules": [
+        {
+            "description": "do not update cake.tool - it is pinned by cake.recipe",
+            "matchFileNames": [".config/dotnet-tools.json"],
+            "matchDepNames": ["cake.tool"],
+            "enabled": false
```
Comment (pascalberger) [inline cake-recipe.json:12]: Why is this not enabled?

**Review by pascalberger — CHANGES_REQUESTED** [review]
`cake-issues.json` could inherit from `cake-recipe` as these repos all use Cake.Recipe

File: cake-recipe.json (hunk 1 above)
Comment (nils-a) [reply cake-recipe.json:12]: So cake.tool will not be updated independently from cake.recipe.
Am I doing it wrong?

File: cake-recipe.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Use Renovate configuration for Cake.Recipe based builds",
+    "extends": [
+        "github>cake-contrib/renovate-presets"
+    ],
+    "packageRules": [
+        {
+            "description": "do not update cake.tool - it is pinned by cake.recipe",
```
Comment (pascalberger) [inline cake-recipe.json]:
```suggestion
            "description": "Do not update Cake.Tool since it is pinned by Cake.Recipe",
```

**Review by pascalberger — APPROVED** [review]
Beside the spelling improvement and that Cake.Issues preset can inherit from this one LGTM

**Comment by nils-a** [comment]
I'll change the Cake.Issues inheritance later today, then I'll merge it in.

**Pull request merged**